packages/orama/src/components/index.ts

```diff
@@ -491,9 +491,22 @@ export async function getSearchablePropertiesWithTypes(index: Index): Promise<Re
   return index.searchablePropertiesWithTypes
 }
 
+function loadNode(node: RadixNode): RadixNode {
+  const convertedNode = radixCreate(node.end, node.subWord, node.key, );
+
+  convertedNode.docs = node.docs;
+  convertedNode.word = node.word;
+
+  for (const childrenKey of Object.keys(node.children)) {
```

Comment:
Why not `const childrenKey in node.children`? AFAIK that should be a bit faster

Reply:
I don't know exactly if is slower or faster.

For [this benchmark](https://measurethat.net/Benchmarks/Show/2375/0/for-in-vs-objectkeys#latest_results_block), `for in` is faster.
For [this other](https://simone-sanfratello.github.io/public/node-bench-iteration/index.html), it is slower.

The only thing I know is `for in` usually causes `deoptmizations` on v8, refs: 
- https://github.com/zhangchiqing/OptimizationKillers/blob/master/README.md
- https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#5-for-in

Reply:
In your benchmark:

```js
for (var i=10000; i > 0; i--) {
  Object.keys(obj).forEach(key => console.log(key));
}
```

there's also an extra function call as a callback for the `forEach` method. This adds slight overhead, so I don't think the benchmark itself is reliable enough.

> https://github.com/zhangchiqing/OptimizationKillers/blob/master/README.md
> https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#5-for-in

These links are interesting and I'd rather follow them, lgtm